source/particle/world.cc

```diff
@@ -1265,18 +1286,28 @@ namespace aspect
         sim->initialize_simulator (this->get_simulator());
       generator->parse_parameters(prm);
 
-      // Create an output object depending on what the parameters specify
-      output.reset(Output::create_particle_output<dim>
-                   (prm));
+      // Remove duplicate output formats specified in the parameter file.
+      auto last = std::unique(output_format_names.begin(), output_format_names.end());
+      output_format_names.erase(last, output_format_names.end());
 
-      // We allow to not generate any output plugin, in which case output is
-      // a null pointer. Only initialize output if it was created.
-      if (output)
+      // Create an output object depending on what the parameters specify
+      if (std::find (output_format_names.begin(),
+                     output_format_names.end(),
+                     "none") == output_format_names.end())
```

Comment:
I think it would make sense to just do 
```
AssertThrow ((std::find (output_format_names.begin(),
                      output_format_names.end(),
                      "none") == output_format_names.end())
                     ||
                     (output_format_names.size() == 1),
                     ExcMessage ("If you specify 'none' for the output format for parameter ...., then this needs to be the "
                                            "only value given."));
```